packages/blade/src/components/Collapsible/_decisions/decisions.md

```diff
@@ -0,0 +1,186 @@
+# Collapsible
+
+Collapsible is used to allow users to toggle the visibility of hidden content within a container.
+
+This document outlines the API of `Collapsible` component. This shares similarities with [Accordion's API](https://github.com/razorpay/blade/pull/1189).
+
+<div align="center">
+  <img src="./collapsible-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Collapsible](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-629874&t=sVxH3DOnx3L3F9rO-0)
+
+## Interactions
+
+Collapsible shares the opening / closing interaction with Accordion:
+
+### Accordion open
+
+https://github.com/razorpay/blade/assets/6682655/1a02d680-1832-4a70-a0a3-1a1f56f2b8b9
+
+### Accordion close
+
+https://github.com/razorpay/blade/assets/6682655/b54eaeb9-7257-4da4-bf3b-89e10785e4f9
+
+## API
+
+Annotated component with props ([figma](https://www.figma.com/file/cX625jYMAX1Roq8hLifFrn/Collapsible-breakdown?type=design&node-id=0-1&t=2fpp9pyvcuMxVTjW-0)):
+
+![Collapsible breakdown](./collapsible-breakdown.png)
+
+```jsx
+import { Collapsible, CollapsibleButton, CollapsibleLink, CollapsibleBody } from '@razorpay/blade';
+
+<Collapsible>
+  <CollapsibleButton>Button</CollapsibleButton>
+  <CollapsibleBody>
+    <Box>Slot</Box>
+  </CollapsibleBody>
+</Collapsible>;
+
+<Collapsible>
+  <CollapsibleLink>Button</CollapsibleLink>
+  <CollapsibleBody>
+    <Box>Slot</Box>
+  </CollapsibleBody>
+</Collapsible>;
+```
+
+> **Note**
+>
+> - `Collapsible` will compose (`CollapsibleButton` or `CollapsibleLink`), `CollapsibleBody`. This ensures all interactions, a11y and animations work seamlessly at user end.
+> - `CollapsibleButton` and `CollapsibleLink` will support a subset of relevant props from `Button` and `Link` respectively.
+
+### Collapsible
+
+| Prop              | Type            | Default     | Description                                                        | Required |
+| ----------------- | --------------- | ----------- | ------------------------------------------------------------------ | -------- |
+| children          | `JSX`           | `undefined` | Composes `CollapsibleButton`, `CollapsibleLink`, `CollapsibleBody` | ✅       |
+| direction         | `bottom`, `top` | `bottom`    | Direction in which the content expands                             |          |
```

Comment:
How will this look in action? Do we have motion or designs for this? 

Like: 

content
button

When clicking the button the content height will animation from bottom to top?

Reply:
Yes, the content will expand which means the trigger will shift down (there are some usecases in the design)